doc/develop/test/twister/hardware-fixtures.rst

```diff
@@ -0,0 +1,56 @@
+Hardware fixture table
+^^^^^^^^^^^^^^^^^^^^^^
+
++----------------------------------+--------------------------------------------------------------------------------------------+
```

Comment:
would this be better that we add one column for the testcase name that use this fixture?